core/contracts/financial-templates/implementation/TokenFactory.sol

```diff
@@ -4,12 +4,17 @@ import "../../common/interfaces/ExpandedIERC20.sol";
 
 
 /**
- * @notice A factory for creating new mintable and burnable tokens.
+ * @title Factory for creating new mintable and burnable tokens.
  */
+
 contract TokenFactory {
     /**
-     * @notice Create a new token and return to the caller. The caller will become the only minter and burner
-     * and the new owner capable of adding new roles.
+     * @notice Create a new token and return to the caller.
+     * @dev The caller will become the only minter and burner and the new owner capable of adding new roles.
+     * @param tokenName used to describe the new token.
+     * @param tokenSymbol short ticker abbreviation of the name. Ideally < 5 chars.
+     * @param tokenDecimals used to delimitate the new token.
```

Comment:
delimitate? Is this a word? Maybe better to reference the "precision", take a look at OZ's description:

```/**
     * @dev Returns the number of decimals used to get its user representation.
     * For example, if `decimals` equals `2`, a balance of `505` tokens should
     * be displayed to a user as `5,05` (`505 / 10 ** 2`).
     *
     * Tokens usually opt for a value of 18, imitating the relationship between
     * Ether and Wei.
     *
     * NOTE: This information is only used for _display_ purposes: it in
     * no way affects any of the arithmetic of the contract, including
     * {IERC20-balanceOf} and {IERC20-transfer}.
     */```